tests/test_tasks/test_task_functions.py

```diff
@@ -156,6 +156,11 @@ def test_get_task_with_cache(self):
         task = openml.tasks.get_task(1)
         self.assertIsInstance(task, OpenMLTask)
 
+    def test_get_task_clustering(self):
```

Comment:
Could you please add a unit test for regression, too.

Reply:
yes I can